packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -38,11 +38,24 @@ BuildContext? _getAncestor(BuildContext context, {int count = 1}) {
 void _focusAndEnsureVisible(
   FocusNode node, {
   ScrollPositionAlignmentPolicy alignmentPolicy = ScrollPositionAlignmentPolicy.explicit,
+  double alignment = 1.0,
+  Duration duration = Duration.zero,
+  Curve curve = Curves.ease,
 }) {
   node.requestFocus();
-  Scrollable.ensureVisible(node.context!, alignment: 1.0, alignmentPolicy: alignmentPolicy);
+  Scrollable.ensureVisible(node.context!, alignment: alignment, alignmentPolicy: alignmentPolicy, duration: duration, curve: curve);
 }
 
+/// Signature for the callback that's called when a traversal policy is
+/// requesting focus.
+typedef RequestFocusCallback = void Function(
```

Comment:
This should probably have a name that reflects that it is part of focus traversal, and not some version of the requestFocus on a focus node. Maybe `TraversalRequestFocusCallback`?